packages/cozy-scanner/src/DocumentQualification.jsx

```diff
@@ -42,11 +40,19 @@ export class DocumentQualification extends Component {
     this.state = {
       selected: { categoryLabel, item },
       filename: this.defaultFilename,
-      hasUserWrittenFileName: false
+      hasUserWrittenFileName: false,
+      overridedThemes: null
     }
     this.textInput = React.createRef()
   }
 
+  componentDidMount() {
```

Comment:
can you add the UNSAFE_ prefix please? Also, why not during the `willMount` ?

Reply:
Indeed, the `UNSAFE_componentWillMount` method is more suitable.

PS: The `componentDid...` and `componentWillUnmount` methods do not have a prefix.

Reply:
> PS: The componentDid... and componentWillUnmount methods do not have a prefix.

Oh. To old to remember that stuff ^^'